mrp_sale_info/models/mrp_production.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antiun Ingenieria S.L. - Javier Iniesta
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields
+
+
+class MrpProduction(models.Model):
+    _inherit = "mrp.production"
+
+    sale_id = fields.Many2one(
+        'sale.order', string='Sale order', readonly=True, store=True,
+        related='move_prod_id.procurement_id.sale_line_id.order_id')
+    partner_id = fields.Many2one(related='sale_id.partner_id',
```

Comment:
Sorry, I do not remember, our module is used as information only, those fields are not functionally used for anything else.